cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -140,43 +140,44 @@ def _handle_exception(
 
     Args:
-        quality_preset ({"base", "low", "medium", "high", "best"}, default = "medium"): An optional preset configuration to control
-            the quality of TLM responses and trustworthiness scores vs. runtimes/costs.
+        quality_preset ({"base", "low", "medium", "high", "best"}, default = "medium"): an optional preset configuration to control
+            the quality of TLM responses and trustworthiness scores vs. latency/costs.
 
-            The "best" and "high" presets return improved LLM responses,
+            The "best" and "high" presets auto-improve LLM responses,
             with "best" also returning more reliable trustworthiness scores than "high".
             The "medium" and "low" presets return standard LLM responses along with associated trustworthiness scores,
-            with "medium" producing more reliable trustworthiness scores than low.
-            The "base" preset will provide a standard LLM response and a trustworthiness score in the lowest possible latency/cost.
+            with "medium" producing more reliable trustworthiness scores than "low".
+            The "base" preset provides a standard LLM response and a trustworthiness score in the lowest possible latency/cost.
 
             Higher presets have increased runtime and cost (and may internally consume more tokens).
             Reduce your preset if you see token-limit errors.
-            Details about each present are in the documentation for [TLMOptions](#class-tlmoptions).
-            Avoid using "best" or "high" presets if you primarily want trustworthiness scores (i.e. are using `tlm.get_trustworthiness_score()` rather than `tlm.prompt()`), and are less concerned with improving LLM responses.
-            These "best" and "high" presets have higher runtime/cost, and are optimized to return more accurate LLM outputs, but not more reliable trustworthiness scores than the "medium" and "low" presets.
+            Details about each present are documented in [TLMOptions](#class-tlmoptions).
+            Avoid "best" or "high" presets if you just want trustworthiness scores (i.e. are using `tlm.get_trustworthiness_score()` rather than `tlm.prompt()`).
+            These "best" or "high" presets can additionally improve the LLM response itself, but do not return more reliable trustworthiness scores than "medium" or "low" presets.
 
-        options (TLMOptions, optional): a typed dict of advanced configuration options.
-        Available options (keys in this dict) include "model", "max_tokens", "num_candidate_responses", "num_consistency_samples", "use_self_reflection",
+        options (TLMOptions, optional): a typed dict of advanced configurations you can optionally specify.
+        Available options (keys in this dict): "model", "max_tokens", "num_candidate_responses", "num_consistency_samples", "use_self_reflection",
```

Comment:
```suggestion
        Available options (keys in this dict) include "model", "max_tokens", "num_candidate_responses", "num_consistency_samples", "use_self_reflection",
```

`:` tends to cause weird formatting issues